code/game/objects/items/misc.dm

```diff
@@ -21,4 +21,11 @@
 	throwforce = 0.0
 	throw_speed = 1
 	throw_range = 20
-	flags = CONDUCT
\ No newline at end of file
+	flags = CONDUCT
+
+/obj/item/weapon/beach_ball/basketball
```

Comment:
Yeah, the fix for that would need the map freeze to end, so... I guess this PR gets to wait a bit